bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -423,3 +423,70 @@ def pythonize_rdataframe(klass):
     from ._rdf_pyz import _PyFilter, _PyDefine
     klass.Filter = _PyFilter
     klass.Define = _PyDefine
+
+
+def _MakeNumpyDataFrame(np_dict):
+    """
+    Make an RDataFrame from a dictionary of numpy arrays
+
+    \param[in] self Always null, since this is a module function.
+    \param[in] pydata Dictionary with numpy arrays
+
+    This function takes a dictionary of numpy arrays and creates an RDataFrame
+    using the keys as column names and the numpy arrays as data.
+    """
+    import ROOT
+    import platform
+
+    cppyy = ROOT.module.cppyy
```

Comment:
Why can't we `import cppyy` directly in this case?

Reply:
That's true, I think it can be done